src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -1411,7 +1411,7 @@
     <value>For determining whether the object contains or not any items, prefer using 'IsEmpty' property rather than retrieving the number of items from the 'Count' property and comparing it to 0 or 1.</value>
   </data>
   <data name="PreferIsEmptyOverCountMessage" xml:space="preserve">
-    <value>Prefer 'IsEmpty' over 'Count' to determine whether the object contains or not any items</value>
+    <value>Prefer 'IsEmpty' over 'Count' to determine whether the object is empty.</value>
```

Comment:
@BrunoJuchli Sorry I missed it, the message string should not end with a period.

Reply:
Yes, this will cause build failure.

Reply:
Also tagging @Jozkee for review.

Reply:
Thank you. I've removed the trailing period in the Message.
I've also done so in the French and in the German translations (even though it did have a trailing period before). I guess that's ok (but the build is not checking translations for correctness in that regard) -- right?
I haven't checked/fixed translations to other languages. I could but then I believe it would probably be best to do this in a separate PR - do you concur?

Reply:
@BrunoJuchli You are right, the analyzer only check the English message, title and description entries. I was thinking about checking the other languages but that would probably require to discuss with native speakers to see what they think. Basically we wrote this analyzer to ensure that there is a consistent style in how messages are shown to the user (at least partially).